Game.cs

```diff
@@ -8,18 +9,500 @@ internal class Game
+                                }
+                                else if (player.inventoryContents().Contains("Shield"))
+                                {
+                                    currentRoom.removeItem("Shield");
+                                }
+                                else if (player.inventoryContents().Contains("Potion"))
+                                {
+                                    currentRoom.removeItem("Potion");
+                                }
+                            }
+                            else if (player.steps >= 16 && player.steps <= 25)
+                            {
+                                if (player.steps == 15)
+                                {
+                                    Console.WriteLine("You have now entered a new room");
+                                    player.addScore(100);
+                                }
+                                currentRoom = new Room("You are in a room with a large fireplace, a grand piano and a large bookshelf\n");
+                                if (player.inventoryContents().Contains("Sword"))
+                                {
+                                    currentRoom.removeItem("Sword");
+                                }
+                                else if (player.inventoryContents().Contains("Shield"))
+                                {
+                                    currentRoom.removeItem("Shield");
+                                }
+                                else if (player.inventoryContents().Contains("Potion"))
+                                {
+                                    currentRoom.removeItem("Potion");
+                                }
+                            }
+                            else if (player.steps >= 26 && player.steps <= 35)
+                            {
+                                if (player.steps == 26)
+                                {
+                                    Console.WriteLine("You have now entered a new room");
+                                    player.addScore(100);
+                                }
+                                currentRoom = new Room("You are in a room with a large bed, a wardrobe and a large mirror\n");
+                                if (player.inventoryContents().Contains("Sword"))
+                                {
+                                    currentRoom.removeItem("Sword");
+                                }
+                                else if (player.inventoryContents().Contains("Shield"))
+                                {
+                                    currentRoom.removeItem("Shield");
+                                }
+                                else if (player.inventoryContents().Contains("Potion"))
+                                {
+                                    currentRoom.removeItem("Potion");
+                                }
+                            }
```

Comment:
A lot of repetition with this if clauses. Would be better to have a `List<Room>` that is initialised as static in the `Game` class, additionally, the Room class could contain a `size` field, that determines how many steps are required before the player enters a new room.

This way you don't have to hard code the values in these if statements but instead could have a `currentRoom` field that tracks what room the player is currently in based on the number of steps they have done. This would be quite a big change to the flow of the code, however.